index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Figma to HTML & CSS</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500;800&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="app">
+      <nav class="nav-bar">
+        <ul class="nav-bar-list">
+            <li class="nav-link"><a class="nav-link-text" href="#">
+             <img src="images/applogo.png" alt="logo"> 
+            </a></li>
+            <li class="nav-link"><a class="nav-link-text" href="#">Home</a></li>
+            <li class="nav-link"><a class="nav-link-text" href="#">Products</a></li>
+            <li class="nav-link"><a class="nav-link-text" href="#">Pricing</a></li>
+            <li class="nav-link"><a class="nav-link-text" href="#">Blogs</a></li>
+            <li class="nav-link"><a class="nav-link-text" href="#">Jobs</a></li>
+            <li class="nav-link"><a class="nav-link-text" href="#">More</a></li>
+            </ul>
+              <ul class="nav-bar-list-right">
+                <li class="nav-link"><a class="nav-link-text" href="#">
+                <img src="images/google-play.png" alt="googl play badge"> 
+                </a></li>
+                <li class="nav-link"><a class="nav-link-text" href="#">
+                <img src="images/app-store-badge.png" alt="app store badge"> 
+                </a></li>
+              </ul>
+      </nav>
+      <section class="hero-section">
+        <div class="hero-content-wrapper">
+          <div class="hero-content">
+            <h1 class="hero-title">
+            Organise projects. <br>
+            Get more done.
+            </h1>
+            <button class="hero-button">
+            Get started
+            </button>
+          </div>
+          <div class="hero-image-container">
+          <img src="images/phone-mockup.png" alt="hero-image">
+          </div>
+        </div>
+        <div class="hero-company-logos">
+          <img src="images/company-logos/airbnb-logo.png" alt="company logo">
+          <img src="images/company-logos/hubspot-logo.png" alt="company logo">
+          <img src="images/company-logos/google-logo.png" alt="company logo">
+          <img src="images/company-logos/microsoft-logo.png" alt="company logo">
+          <img src="images/company-logos/walmart-logo.png" alt="company logo">
+          <img src="images/company-logos/fedex-logo.png" alt="company logo">
+        </div>
```

Comment:
*Suggestion*

- [ ] Please move the `hero-compnay-logos` into a diff PR it's more of a section on it's own.